gadget-container/entrypoint.sh

```diff
@@ -59,7 +59,7 @@ if grep -q '^1:name=systemd:.*/crio-[0-9a-f]*\.scope$' /proc/self/cgroup > /dev/
     CRIO=1
 fi
 
-# In the gadget image, /usr/src is a symlink to /host/usr/src (see gadget.Dockerfile).
+# In the gadget image, /usr/src is a symlink to /host/usr/src (see Dockerfile).
```

Comment:
Why did you remove the prefix here? There isn't a "Dockerfile" file. Do you mean to point to any of the Dockerfiles? I think maybe gadget.*.Dockerfile would be clearer in that case? Otherwise, if I'm reading this and then find that there is no Dockerfile, I'd assume something got broken in between.

Reply:
I wanted to make it generic and valid for both Dockerfiles, changed to `gadget.*.Dockerfile`.